dataprovider-retrofit/src/main/java/ch/srg/dataProvider/integrationlayer/request/parameters/IlType.kt

```diff
@@ -7,4 +7,4 @@ import ch.srg.dataProvider.integrationlayer.data.Type
  * <p>
  * License information is available from the LICENSE file.
  */
-class ILType(type: Type) : IlParam(type.toString().lowercase())
\ No newline at end of file
+class ILType(type: Type) : IlParam(type.toString().lowercase())
```

Comment:
⚠️ **[detekt]** <detekt.MatchingDeclarationName> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>The file name 'IlType' does not match the name of the single top-level declaration 'ILType'.